example/ingest_books_only.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+import json
+import os
+import re
+import tarfile
+from concurrent.futures import ProcessPoolExecutor, as_completed
+from pathlib import Path
+from urllib import request
+from urllib.error import (
+    URLError,
+)
+from typing import (
+    Callable,
+    Dict,
+)
+
+
+# turn on/off diagnostic information
+LOG_LEVEL = 0
```

Comment:
please use the `logging` module from the python stdlib instead of the `if LOG_LEVEL > ...` + `print` pattern, makes for much less verbose/nested code.